demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+export const LocalMediaStreamMetrics: React.FC = () => {
+  const audioVideo = useAudioVideo();
+  const localAttendeeId = (audioVideo as any).audioVideoController
+    ?.realtimeController?.state?.localAttendeeId;
+  const {
+    audioPacketsSentFractionLossPercent,
+    audioPacketsReceivedFractionLossPercent,
+    availableIncomingBandwidth,
+    availableOutgoingBandwidth,
+    videoStreamMetrics
+  } = useMediaStreamMetrics();
+  const localVideoStreamMetrics = videoStreamMetrics[localAttendeeId];
+  const ssrcArray = localVideoStreamMetrics
+    ? Object.keys(localVideoStreamMetrics)
+    : [];
+  const isLocalAudioActive =
+    audioVideo &&
+    audioPacketsSentFractionLossPercent !== null &&
+    audioPacketsReceivedFractionLossPercent !== null;
+  const isLocalVideoActive = audioVideo && ssrcArray.length !== 0;
```

Comment:
Should we check `audioVideo` validity up earlier after `useAudioVideo()` itself?

Reply:
sure, will rearrange.

Reply:
Can you please check this again, looks like still not resolved?

Reply:
I am not sure if I get your point correctly. I already move the  
```
 const isLocalAudioActive =
  audioVideo &&
  audioPacketsSentFractionLossPercent !== null &&
  audioPacketsReceivedFractionLossPercent !== null;
```
up to make is below the code:
```
  const audioVideo = useAudioVideo();
  const {
    audioPacketsSentFractionLossPercent,
    audioPacketsReceivedFractionLossPercent,
    availableIncomingBandwidth,
    availableOutgoingBandwidth,
    videoStreamMetrics
  } = useMediaStreamMetrics();
```